src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age=" + age + ", name='" + name + '\''
+                + ", ownerName='" + ownerName + '\'' + '}';
```

Comment:
```suggestion
                + "age=" + age 
                + ", name='" + name + '\''
                + ", ownerName='" + ownerName + '\'' + '}';
```
this is more beautiful code